flow_generator.py

```diff
@@ -0,0 +1,132 @@
+import xml.etree.ElementTree as ET
+import random
+import shutil as sh
+
+
+def add_vehicle_flows(original_xml_file_path, text_xml_file, inset_flows, new_index, level):
```

Comment:
add parameters type and return type
Example: def add_vehicle_flows(... , new_index: int, level: str) -> None